packages/@aws-cdk/aws-ec2/lib/vpc-flow-logs.ts

```diff
@@ -401,6 +401,115 @@ export enum FlowLogMaxAggregationInterval {
 
 }
 
+/**
+ * The following table describes all of the available fields for a flow log record.
+ */
+export enum LogFormatField {
+  /**
+    * The VPC Flow Logs version.
+    */
```

Comment:
```suggestion
  /**
   * The VPC Flow Logs version.
   */
```

nit - can you fix all the formatting of these docstrings so they are consistent.